scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/bsp/extension/ScalaBuildTargetClassifier.scala

```diff
@@ -17,7 +23,7 @@ private class ScalaBuildTargetClassifier extends BuildTargetClassifierExtension
   override def calculateBuildTargetPath(buildTarget: String): util.List[String] = {
     buildTarget match {
       case buildTargetFileRegex(_, path, _) =>
-        path.split("/").toList.asJava
+        path.split("/").toList.takeRight(2).asJava
```

Comment:
What if I had nested build target (myNestedProject) like project/subProjects/myNestedProject, then it wouldn't work correctly because path would get cut after project, right?